integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -78,59 +80,20 @@ void init() {
     sql("USE " + getDefaultDatabase());
   }
 
+  @AfterAll
+  void cleanUp() {
+    sql("USE " + getCatalogName());
+    getDatabases()
+        .forEach(
+            databaseName -> sql(String.format("DROP DATABASE IF EXISTS %s CASCADE", databaseName)));
+  }
+
   @Test
   void testLoadCatalogs() {
     Set<String> catalogs = getCatalogs();
     Assertions.assertTrue(catalogs.contains(getCatalogName()));
   }
 
-  @Test
-  void testCreateAndLoadSchema() {
```

Comment:
Totally I prefer to keep schema tests in `SparkCommonIT`, because this implement is same. why move them to SparkXXCatalogIT?

Reply:
seems not resoved

Reply:
> seems not resovled

resolved.